src/c++/vdo/fake/linux/bio.h

```diff
@@ -11,7 +11,22 @@
 #define __LINUX_BIO_H
 
 /* struct bio, bio_vec and BIO_* flags are defined in blk_types.h */
+#include <linux/version.h>
+#undef VDO_USE_ALTERNATE
```

Comment:
fake/linux/bio.h is only used for user-mode builds, isn't it? Won't both versions from userLinux/uds be available? Maybe for this file we can just switch to the new name and not conditionalize it?

Reply:
Turns out this doesn't work. We have to conditionalize so that we don't end up pulling both versions when a file includes both numeric.h and bio.h (which is a lot of files).